contrib/syslog-ng-debun

```diff
@@ -870,6 +870,16 @@ fhs_set_unix () {
 	:
 }
 
+save_config_file() {
```

Comment:
The `acquire_syslog_config` already saves the configuration (from disk to debun).

Reply:
ok.